get_data_for_tests.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do not include this file in the PR

Reply:
Use `for-testers` branch to get data only.

But commit on a branch from master, and target master.